### PR TITLE
DOCS-7951 made edits

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc [hunk 1]
```diff
@@ -21,15 +21,23 @@ To apply policies:
 . Navigate to *API Manager* and select your API instance.
 . From the left navigation, *select Policies > Apply New Policy*:
 +
-image::select-policies-for-service-mesh.png[]
+.The diagram shows how to apply a policy to your API.
+image::select-policies-for-service-mesh.png[Applying Policies to APIs]
++
+<1> The page where you specify a policy for your API is displayed.
+<2> Select any one of the policies to apply.
```
Comment (dhirsch1) [inline modules/ROOT/pages/manage-apis-with-service-mesh.adoc]: Suggestion: change "any one of the policies" to "a policy."

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc [hunk 2]
```diff
@@ -21,15 +21,23 @@ To apply policies:
...
 * *JWT Validation* 
 * *Rate Limit* 
 * *Rate Limit SLA-Based* 
-
-image::apply-client-enforcement-policy-service-mesh.png[]
++
+.The diagram shows how to apply the Client ID Based Policy.
+image::apply-client-enforcement-policy-service-mesh.png[Apply Client ID Based Policy]
++
+<1> The name of the policy is displayed at the top of the policy page.
```
Comment (dhirsch1) [inline modules/ROOT/pages/manage-apis-with-service-mesh.adoc:39]: I notice that the list after this diagram has roman numerals, but the one after the next one has letters. These should be consistent.

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc [hunk 3]
```diff
@@ -21,15 +21,23 @@ To apply policies:
...
+<2> Select any one of the policies to apply.
 +
 . Select any of the following policies to secure your APIs: 
 * *Client ID Enforcement* 
 * *JWT Validation* 
 * *Rate Limit* 
 * *Rate Limit SLA-Based* 
-
-image::apply-client-enforcement-policy-service-mesh.png[]
++
```
Comment (dhirsch1) [inline modules/ROOT/pages/manage-apis-with-service-mesh.adoc:35]: This diagram is at a lower level in the list than the previous one and I'm not seeing a reason for that.

**Review by dhirsch1 — APPROVED** [review]
Approved with comments

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-apis-with-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-apis-with-service-mesh.adoc:35]: Not sure how I can change it, so keeping it as is for now.

**Pull request merged**